examples/02-plot/movie_glyphs.py

```diff
@@ -0,0 +1,70 @@
+"""
+.. _movie_glyph_example:
+
+Save a Movie Using Glyphs
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Create an animated GIF by generating glyphs using :func:`glyph()
+<pyvista.DataSet.glyph>` using :func:`pyvista.Sphere`.
+
+"""
+# sphinx_gallery_thumbnail_number = 1
+
+import numpy as np
+
+import pyvista as pv
+
+###############################################################################
+# Create sphere glyphs
+# ~~~~~~~~~~~~~~~~~~~~
+
+x = np.arange(-10, 10, 1, dtype=float)
+y = np.arange(-10, 10, 1, dtype=float)
+x, y = np.meshgrid(x, y)
+r = np.sqrt(x**2 + y**2)
+z = (np.sin(r) + 1) / 2
+
+# Create and structured surface
+grid = pv.StructuredGrid(x, y, z)
+grid.point_data['size'] = z.ravel()
+
+# generate glyphs based on the size of the
```

Comment:
I'm not sure if this is what you meant, but the current version needs clarifying.

```suggestion
# generate glyphs with varying size
```